sources/zendesk/__init__.py

```diff
@@ -28,13 +29,16 @@
 def zendesk_talk(
     credentials: TZendeskCredentials = dlt.secrets.value,
     incremental_start_time: Optional[pendulum.DateTime] = DEFAULT_START_DATE,
+    incremental_end_time: Optional[pendulum.DateTime] = None,
 ) -> Iterable[DltResource]:
     """
     Retrieves data from Zendesk Talk for phone calls and voicemails.
 
     Args:
         credentials (TZendeskCredentials): The credentials for authentication. Defaults to the value in the `dlt.secrets` object.
         incremental_start_time (Optional[pendulum.DateTime]): The start time for incremental loading. Defaults to DEFAULT_START_DATE.
+        incremental_end_time: Optionally load records only until this date/time.
```

Comment:
also rename. and give the correct description: `The end time of the range for which to load data. If end time is not provided, the incremental loading will be enabled and after initial run, only new data will be retrieved"